app/playlist.js

```diff
@@ -0,0 +1,49 @@
+function Playlist(id,title,img,about){
```

Comment:
create Song and Playlist objects from received JSON. Use these Song and Playlist object everywhere in the app instead of the plain json